templates/beta/docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
       - ipfs
   devnet:
     image: aragon/geth-devnet
+    volumes:
+      - "./ethereum-data/:/root/.ethdata"
```

Comment:
I think this local path should be somewhere else, or even a variable
```
/home/bingen/workspace/aragon-apps/templates/beta/% ll
total 740K
drwxrwxr-x   9 bingen bingen 4,0K mar 13 16:22 ./
drwxrwxr-x   6 bingen bingen 4,0K mar  2 16:30 ../
-rw-rw-r--   1 bingen bingen 9,1K mar  8 19:37 .abis.json
drwxrwxr-x   2 bingen bingen 4,0K mar 13 15:55 contracts/
drwxrwxr-x   4 bingen bingen 4,0K mar  2 14:25 coverage/
-rw-rw-r--   1 bingen bingen 5,9K mar  5 21:26 coverage.json
-rw-rw-r--   1 bingen bingen  725 mar 13 15:47 docker-compose.yml
-rw-rw-r--   1 bingen bingen  496 mar 13 11:59 Dockerfile
-rw-rw-r--   1 bingen bingen   19 mar 13 11:59 .dockerignore
-rw-rw-r--   1 bingen bingen   49 mar 13 11:59 .env
drwxr-xr-x   4 root   root   4,0K mar 13 15:56 ethereum-data/  <---------------------- !!!
-rw-rw-r--   1 bingen bingen   26 mar 13 15:55 .gitignore
-rw-rw-r--   1 bingen bingen  474 mar 13 11:59 index.js
drwxrwxr-x   2 bingen bingen 4,0K mar 13 15:55 migrations/
drwxrwxr-x 803 bingen bingen 120K mar  8 19:37 node_modules/
-rw-rw-r--   1 bingen bingen 1,9K mar 13 16:22 package.json
-rw-rw-r--   1 bingen bingen 392K mar 13 15:55 package-lock.json
-rw-rw-r--   1 bingen bingen 4,4K mar 13 15:47 readme.md
drwxrwxr-x   2 bingen bingen 4,0K mar 13 15:55 scripts/
-rw-rw-r--   1 bingen bingen  305 mar 13 15:51 .solcover.js
-rw-rw-r--   1 bingen bingen    1 mar  7 19:48 .soliumignore
-rw-rw-r--   1 bingen bingen  589 mar  7 19:48 .soliumrc.json
drwxrwxr-x   3 bingen bingen 4,0K mar 13 16:31 test/
-rw-rw-r--   1 bingen bingen  142 feb 23 18:03 truffle.js
```
And then trying to build I got:
```
/home/bingen/workspace/aragon-apps/templates/beta/% docker-compose stop && npm run docker:build && docker-compose up deploy; echo $?
Stopping beta_devnet_1 ... done
Stopping beta_ipfs_1 ... done

> @aragon/templates-beta@1.1.1 docker:build /home/bingen/workspace/aragon-apps/templates/beta
> docker build -t aragon/templates-beta -f Dockerfile ../..

error checking context: 'can't stat '/home/bingen/workspace/aragon-apps/templates/beta/ethereum-data/geth''.
```